web/src/components/ToggleButton.tsx

```diff
@@ -53,14 +51,14 @@ export default function ToggleButton({
             key={`group-item-${key}`}
             value={option.value}
             onClick={() => onToggle(option.value)}
-            className={`
-       inline-flex h-[26px] w-full  items-center whitespace-nowrap rounded-full px-4 ${fontSize} ${
+            className={twMerge(
+              'inline-flex h-[26px] w-full items-center whitespace-nowrap rounded-full px-4 text-xs',
```

Comment:
```suggestion
            className={twMerge(
              'inline-flex h-[26px] w-full items-center whitespace-nowrap rounded-full px-4 text-sm',
```
This was sm before, should we keep it as such?
Before:
![image](https://user-images.githubusercontent.com/30777521/224678437-4a00e685-83eb-453d-8608-00949872df12.png)
After:
![image](https://user-images.githubusercontent.com/30777521/224678491-587535a2-6d75-4b8b-ba29-2acf8aa520a2.png)

Reply:
The figma file was even smaller, I think xs is pretty good but it's up to @rasmusjeppsen (the netlify preview should help)

Reply:
Hmm, I'm not sure it's a great combo to both make these smaller and make them bold. It might work fine for higher resolution screens (but even then, the text is really small) but on lower resolution screens things will become almost merged at that size and would make things hard to read, especially at first glance.

I think just bumping the size to what it was before should take care of it.

Reply:
Looks great! 

**A few feedback points:**
Great points of feedback, Viktor. I agree and suggest that:
1. We should make the not-toggled option "regular" and only the toggled option "bold" consistent with the historical view toggles.
2. After seeing it live on the Netlify preview, I think we should bump up the font size to 12px to match the historical view toggles and ensure accessibility is optimal in terms of readability. In other words, bumping the size to what it was before, as Viktor suggests.

PS. I'm aware that my Figma file had some challenging font sizes! Great job Tony ⚡️